securesystemslib/interface.py

```diff
@@ -237,11 +237,20 @@ def generate_and_write_rsa_keypair(filepath=None, bits=DEFAULT_RSA_KEY_BITS,
 
 
 def import_rsa_privatekey_from_file(filepath, password=None,
-    scheme='rsassa-pss-sha256'):
+    scheme='rsassa-pss-sha256', prompt=False):
```

Comment:
NBD: Has there been some discussion about whether or not False should be the default prompt value used?

- TUF tutorial text -- and some code, probably -- will need to change.
- (What are the security implications of passing the passphrase in through a function call in the interpreter rather than using the prompt functionality from `getpass`?)